tests/product/test_collect.py

```diff
@@ -294,3 +274,33 @@ def test_collect_logs_server_not_installed(self):
                            self.PA_ONLY_CLUSTER)
         self.upload_topology()
         self._assert_no_logs_downloaded()
+
+    def test_collect_logs_multiple_server_logs(self):
+        self.setup_cluster(NoHadoopBareImageProvider(),
+                           self.STANDALONE_PRESTO_CLUSTER)
+        self.run_prestoadmin('server start')
+        self.cluster.write_content_to_host('#!/bin/bash\n touch '
+                                           '/var/log/presto/server.log-2',
+                                           '/opt/prestoadmin/script.sh',
+                                           self.cluster.master)
+        self.run_prestoadmin('script run /opt/prestoadmin/script.sh')
+        actual = self.run_prestoadmin('collect logs')
+
+        expected = 'Downloading logs from all the nodes...\n' + \
+                   'logs archive created: ' + OUTPUT_FILENAME_FOR_LOGS + '\n'
+        self.assertLazyMessage(lambda: self.log_msg(actual, expected),
+                               self.assertEqual, actual, expected)
+
+        downloaded_logs_location = path.join(TMP_PRESTO_DEBUG, 'logs')
+        self.assert_path_exists(self.cluster.master,
+                                downloaded_logs_location)
+
+        for host in self.cluster.all_internal_hosts():
+            host_log_location = path.join(downloaded_logs_location,
+                                          host)
+            self.assert_path_exists(self.cluster.master,
```

Comment:
I guess we're asserting that we get server.log. Maybe we should just touch it as well in case the code that gets us to this point changes in a way that we don't get a real server.log file?